pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -3157,12 +3157,22 @@ void GSRendererHW::Draw()
 					        (!no_rt && m_cached_ctx.FRAME.FBP == m_cached_ctx.ZBUF.ZBP && !PRIM->TME && zm == 0 && (fm & fm_mask) == 0 && m_cached_ctx.TEST.ZTE) ||
 					        // No color or Z being written.
 					        (no_rt && zm != 0);
-					if (no_rt && no_ds)
-					{
-						GL_INS("HW: Late draw cancel because no pixels pass alpha test.");
-						CleanupDraw(true);
-						return;
-					}
+				}
+				else
+				{
+					no_rt = no_rt || (!IsRTWritten() && !m_cached_ctx.TEST.DATE);
```

Comment:
V.Small thing I'd swap around the date and RTWritten conditions, since testing DATE is going to be quicker